libs/datamodel/connectors/dml/src/field.rs

```diff
@@ -374,7 +374,7 @@ impl RelationField {
 
         match self.referential_arity {
             _ if !self.emulates_referential_actions.unwrap_or(false) => Cascade,
-            FieldArity::Required => Restrict,
+            FieldArity::Required => NoAction,
```

Comment:
I don't understand this line (can't relate it to the rest of the PR)

Reply:
So we do not support `onUpdate: Restrict` on emulations yet. We actually have nothing supported for `onUpdate` yet, and `NoAction` there does actually what it says: nothing. This means we allow only that, and the default should reflect this even though the change here has no actual meaning.